src/games/Game.vue

```diff
@@ -0,0 +1,80 @@
+<script>
+import ElementNav from './elements/ElementNav.vue'
+import EditElement from './elements/EditElement.vue'
+
+const save = s => localStorage.setItem('elements', JSON.stringify(s))
+const load = () => JSON.parse(localStorage.getItem('elements') || false) || [
+  {
+    slug: 'player-rovs',
+    title: 'ROVs',
+    markdown: `# ROVs\n\nThe core game mechanic is driving remotely operated vehicles around a two-dimensional map.\n`
+  },
+  {
+    slug: 'player-views',
+    title: 'Views',
+    markdown: 'Views are the actual visuals of the game, and have types like sonar or something.'
+  },
+  {
+    slug: 'defenses',
+    title: 'ROV Defense Upgrades',
+    markdown: 'You can get upgrades for your robots.\n\nSometime soon this will support markdown.\n'
+  }
+]
+
+export default {
+  name: 'Game',
+  components: {
+    ElementNav,
+    EditElement
+  },
+  props: ['game', 'slug'],
```

Comment:
Let's get in the habit of using the more verbose form for props. Sure, it's type checking, but there's almost no effort involved in adding it, and it's useful inline documentation. Information on the syntax can be found [here](http://devdocs.io/vue~2/api/index#props).